oauth/authentication-oauth2.0.md

```diff
@@ -44,8 +44,8 @@ POST https://app.solvexia.com/oauth/token
 
 | Name | Type |Description |
 | ------------- |------------- | -------------|
-|client_id|`string`|REQUIRED. Client id that you received when you created an application in SolveXia.|
-|client_secret|`string`|REQUIRED. Client secret that you received when you created an application in SolveXia.|
+|client_id|`string`|REQUIRED. Client id that you've received when you created an application in SolveXia.|
```

Comment:
I think "you" is fine, or maybe "you would have". Also applies to the line below.

Reply:
Thanks, fixed.